libr/core/cmd_anal.c

```diff
@@ -3937,7 +3942,7 @@ R_API void r_core_af(RCore *core, ut64 addr, const char *name, bool anal_calls)
 		__add_vars_sdb (core, fcn);
 	} else {
 		if (core->anal->verbose) {
-			eprintf ("Warning: Unable to analyze function at 0x%08"PFMT64x"\n", addr);
+			R_LOG_WARN ("Warning: Unable to analyze function at 0x%08"PFMT64x"\n", addr);
```

Comment:
I'm finding it a little difficult to know when to use each of the R_LOG_ functions but once I figure it out then I can start working through the rest of the files and replace the eprintf's

Reply:
Try to make small contributions then. So we can discuss each of them before doing a big change thats harder to review and maintain.

i would left the help messages as eprintf because some of them are printed when passing wrong arguments to a command, and that needs some more carefully review.

anyway its an important step forward towards routing the error messages and improve the help message for more commands.